### ISSUE TITLE
New HTMLText is added to the top-left corner of the webpage, is visible and can be interacted with

### ISSUE THREAD

**Issue by ZKtheMAN** [pixijs/html-text#35]
Using PixiJS 7, recently updated the text-html package from NPM, ran into this issue. Any HTMLText that gets created is put onto the webpage, is visible, and can be interacted with. I can highlight it and copy it, for example.

![Picture of a PixiJS instance with an HTMLText on the stage; the HTMLText appears twice, once on the stage and once outside the borders of the stage. The text outside the stage is highlighted.](https://user-images.githubusercontent.com/9014806/209646114-504e3c62-2dbd-4f51-a7fc-6e6e87e37a46.png)

Multiple HTMLText on the stage will add them all to the top-left corner of the website. This gets very weird.

![Picture of another PixiJS instance. A box of choices is on the stage, and each chose is an HTMLText instance. Each of them also appears over each other in the top-left corner. The text for the fifth choice, "5. Signing the same old song", is highlighted in the top left corner.](https://user-images.githubusercontent.com/9014806/209646411-d1ed38a8-d477-4707-ab2b-4474d7760dbb.png)

A minimum reproduction example (this generated the result in the first image):

```typescript
import { Application } from 'pixi.js';
import { HTMLText } from '@pixi/text-html';

const canvasElement = document.getElementById("pixi-canvas") as HTMLCanvasElement;

const options = {
	view: canvasElement,
	resolution: window.devicePixelRatio || 1,
	autoDensity: true,
	backgroundColor: "skyblue",
	width: 1280,
	height: 720,
};

const app = new Application(options);

app.stage.addChild(new HTMLText("HTMLText gets added to the DOM, is visible, can be interacted with", {
	fill: "white",
}))
```

The fill being white isn't necessary. Black HTMLText will still get added to the top left corner, even if I can't see it.

This issue arose on Chrome and Firefox on both Windows and Android.

**Comment by lykl** [comment]
I have a similar problem.
In addition to the fact that text is displayed in the upper left corner, I found in devtool that svg text is repeatedly refreshed, while text is not displayed on the canvas.
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/42017216/210180985-3a7f5d5c-92ad-4a30-ac94-0543d75a53be.png">
Does this plugin support pixi 7.x?
pixi version: 7.0.4
html-text version: 3.2.0

**Comment by bigtimebuddy** [comment]
Will take a look this week. Still getting back from holiday.

**Comment by jdrstx** [comment]
I'm experiencing this too. My temporary workaround is to add the following css class to my top-level html file:

div[data-pixi-html-text="1"] {
  visibility: hidden;
}

**Comment by lykl** [comment]
> I'm experiencing this too. My temporary workaround is to add the following css class to my top-level html file:
> 
> div[data-pixi-html-text="1"] { visibility: hidden; }

This is a good solution, but not enough, since the element blocks the mouse, it's best to add `pointer-events:none;`

**Comment by bigtimebuddy** [comment]
This is fixed in v3.2.1, you don't need that CSS hack anymore.
